test-pipeline.yml

```diff
@@ -1,6 +1,23 @@
 name: Example Deployment
-application: example     # should match application created in spinnaker
-triggers:                # list of triggers (currently only jenkins is supported)
+application: example
+
+disableConcurrentExecutions: true
+keepQueuedPipelines: true
+description: This pipeline deploys some sweet code
+
+notifications:
+  - address: "#launchpad"
```

Comment:
does this allow meta information like version/git hash/etc?

Reply:
Yeah, the messages can use spinnaker expression syntax.